pxtlib/tutorial.ts

```diff
@@ -22,6 +22,12 @@ namespace pxt.tutorial {
             simThemeJson
         } = computeBodyMetadata(body);
 
+        // For python HOC, hide the toolbox (we don't support flyoutOnly mode).
+        if (pxt.BrowserUtils.useOldTutorialLayout() && language === "python" && metadata.flyoutOnly) {
+            metadata.flyoutOnly = false;
```

Comment:
I think this is breaking the change that I made recently for getting rid of the icons for block snippets for `flyoutOnly` tutorials, based on one of the screenshots you included. To get rid of the icon, I'm checking that the metadata has `flyoutOnly` to true.. 
https://github.com/microsoft/pxt/blob/4f68e8bf51adffe42186b353a01004833e9748d3/webapp/src/marked.tsx#L387
Maybe add, on this line, a check for if `hideToolbox` is true as well, because that would also mean that the tutorial wouldn't have categories.

Reply:
Good catch. Made your suggested change and updated the screenshot.